content/api/hub/architecture_singed_webhooks.md

```diff
@@ -0,0 +1,27 @@
+title = "Signed Webhooks"
+template = "render_hub_content_body"
+date = "2024-04-03T13:50:00Z"
+content-type = "text/plain"
+tags = ["webhooks", "rust", "python", "architecture"]
+
+[extra]
+author = "Thorsten Hans"
+type = "hub_document"
+category = "Architecture"
```

Comment:
Is this field an array? If so, could we also include category type "Component"?

Reply:
Check various other files, looks like it isn't an array

Reply:
It currently is not an array but if there is a need for it, I am happy to look into it.